### DISCUSSION TITLE
Signs in to GitHub with another e-mail address than the one you are signing in with

### DISCUSSION THREAD

**Discussion by ghost** [community/community#126401] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have a github user with my personal email address and one with my work email. When I log in with the work email it opens my personal github instead of the work user even if I log in with the work email. How can I access the work github ?

**Comment by davevad93** [comment]
Hi @Natukka ,

If I understood correctly, you have 2 different accounts, one for personal use and the other for work, all you have to do for logging in with your work account is to enter the username (and password) instead of the email.

**Reply by ghost** [reply]
Yes, I have 2 different accounts. I tried to do as you suggested but it still logs in with my personal account, even if I log in with the work email and so..
